### PR TITLE
update venus config

### PR DESCRIPTION
**Pull request by smartdev51**
- update venus config with mainnet&testnet abis, addresses, block numbers, etc.

### CODE REVIEW

**Comment by brightdev33** [comment]
@fengliu93 lets add all new vToken informations on mainnet and testnet.

**Comment by fengliu93** [comment]
okay

**Comment by fengliu93** [comment]
@brightdev33 , just added vToken informations on mainnet and testnet.

File: networks/mainnet.json [hunk 1]
```diff
@@ -17,6 +18,14 @@
...
+      "DOT": "0x7083609fCE4d1d8Dc0C979AAb8c869Ea2C873402",
+      "LINK": "0xF8A0BF9cF54Bb92F17374d9e9A321E6a111a51bD",
+      "DAI": "0x1AF3F329e8BE154074D8769D1FFa4eE058B1DBc3",
+      "FIL": "0x0D8Ce2A99Bb6e3B7Db580eD848240e4a0F9aE153",
+      "BETH": "0x250632378E573c6Be1AC2f97Fcdf00515d0Aa91B",
+      "ADA": "0x3EE2200Efb3400fAbB9AacF31297cBdD1d435D47",
+      "DOGE": "0xbA2aE424d960c26247Dd6c32edC70B295c744C43",
       "vUSDC": "0xecA88125a5ADbe82614ffC12D0DB554E2e2867C8",
       "vUSDT": "0xfD5840Cd36d94D7229439859C0112a4185BC0255",
       "vBUSD": "0x95c78222B3D6e262426483D42CfA53685A67Ab9D",
```
Comment (brightdev33) [inline networks/mainnet.json:31]: @fengliu93 thanks.
we also need vToken addresses for new assets.

File: networks/mainnet.json [hunk 2]
```diff
@@ -49,6 +59,14 @@
...
       "LTC": 326301,
       "XRP": 326495,
+      "BCH": 4731457,
+      "DOT": 6037801,
+      "LINK": 4395407,
+      "DAI": 5385246,
+      "FIL": 4380689,
+      "BETH": 4754650,
+      "ADA": 6207589,
+      "DOGE": 6187851,
```
Comment (brightdev33) [inline networks/mainnet.json:69]: we also need vToken addresses for new assets.

**Review by xioamal — APPROVED** [review]

**Pull request closed without merging**